id/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE_DATA_SYNC" />
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
-    <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.WAKE_LOCK"  tools:replace="true" />
```

Comment:
Shouldn't this be `tools:replace="android:name"`

Reply:
Could be `tools:node="replace"`?

Reply:
In my latest commit, I removed the maxSdkVersion property that is causing the problem